src/components/ListCardPost.vue

```diff
@@ -0,0 +1,205 @@
+<script lang="ts">
+// libraries
+import { defineComponent, computed } from 'vue';
+import { Screen } from 'quasar';
+import { Swiper, SwiperSlide } from 'swiper/vue';
+import { Navigation, A11y } from 'swiper/modules';
+
+// components
+import CardPost from './CardPost.vue';
+
+// types
+import { CardPost as CardPostType, Link } from './types';
+
+export default defineComponent({
+  name: 'ListCardPost',
+  components: {
+    Swiper,
+    SwiperSlide,
+    CardPost,
+  },
+  props: {
+    title: {
+      type: String,
+      required: true,
+    },
+    cards: {
+      type: Array as () => CardPostType[],
+      required: true,
+    },
+    button: {
+      type: Object as () => Link,
+      required: false,
+    },
+  },
+  setup() {
+    const isLargeScreen = computed(() => {
+      return Screen.gt.sm;
+    });
+
+    const buttonWidth = computed(() => {
+      return isLargeScreen.value ? 'auto' : '100%';
+    });
+
+    return {
+      buttonWidth,
+      modules: [Navigation, A11y],
+    };
+  },
+});
+</script>
+
+<template>
+  <!-- Component displaying a slider with news -->
+  <!-- Internal Figma link: https://www.figma.com/file/L8dVREySVXxh3X12TcFDdR/Do-pr%C3%A1ce-na-kole?type=design&node-id=4858%3A105611&mode=design&t=x3DpoanmIFk5i6MU-1 -->
+  <div class="relative-position" data-cy="card-list-post">
+    <!-- Title -->
+    <h2
+      class="text-h6 q-mt-none text-weight-semibold"
+      data-cy="card-list-post-title"
+    >
+      {{ title }}
+    </h2>
+    <!-- Swiper for news cards -->
+    <swiper
```

Comment:
According Vue Swipper [doc](https://swiperjs.com/vue): _Swiper Vue components will likely to be removed in future versions. It is recommended to migrate to [Swiper Element](https://swiperjs.com/element) instead_. 

Your Swipper overriding style looks like a little bit complicated. Is there another simpler way how to do it?

https://stackoverflow.com/questions/65590148/swiperjs-how-do-you-style-the-pagination-bullets

Reply:
Updated.